src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " start working!");

```

Comment:
```suggestion
        System.out.println("Buldozzer " + getName() + " start working!");

```

lets specify class name here, so methods doWork() and stopWork() will be different for all machine classes